testkit/src/main/java/org/apache/calcite/test/DiffRepository.java

```diff
@@ -900,18 +903,21 @@ String filter(
   }
 
   /** Cache key. */
-  private static class Key {
+  protected static class Key {
     private final Class<?> clazz;
     private final DiffRepository baseRepository;
     private final Filter filter;
     private final int indent;
-
+    private URL refFile;
+    String refFilePath;
```

Comment:
To be honest I like the newline after all the variables and before the `Key() {}`